src/lwip/app_ethernet.c

```diff
@@ -26,13 +26,18 @@
 #include "FreeRTOS.h"
 #include "app_ethernet.h"
 #include "ethernetif.h"
+#include "string.h"
 #include "task.h"
 
+struct netif gnetif;
+TaskHandle_t link_handle;
+TaskHandle_t dhcp_handle;
+
 /*Static IP ADDRESS: IP_ADDR0.IP_ADDR1.IP_ADDR2.IP_ADDR3 */
 #define IP_ADDR0 ((uint8_t)192U)
```

Comment:
I think we should define default static IP, gateway and netmask as part of a firmware defaults header file. Out of scope for now but something to note for the future.